nemo_skills/dataset/frontierscience-olympiad/prepare.py

```
@@ -0,0 +1,100 @@
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--split",
+        default="all",
+        choices=["all"] + SUBJECTS,
+        help="Dataset split to process (all/chemistry/biology/physics).",
+    )
+    args = parser.parse_args()
+
+    # Load the FrontierScience olympiad dataset directly from HuggingFace
+    print(f"Downloading FrontierScience olympiad dataset from {OLYMPIAD_URL}...")
+    response = requests.get(OLYMPIAD_URL)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)